_includes/journals/volume1/issue1/paper1.html

```diff
@@ -0,0 +1,30 @@
+
+<h1>{{page.title}}</h1>
+
+<strong>Author(s): {{page.author}}</strong> <br>
+<strong><a target="_blank" href="{{paper.link}}">Get PDF</a></strong>
+
+<div class="article-parents col-md-12">
+    <div class="row">
+        <ul class="small list-inline col-md-12 article-journal-name" style="vertical-align:top">
+            <li style="margin: 0;"><strong style="font-size: larger;">Journal of LatinX in AI (LXAI)</strong></li>
+            <li>{{page.grand_parent}},</li>
+            <li><a target="_blank" href="#">{{page.parent}}</a>,</li>
+            <li>{{page.pagesnumber}}</li>
+            <li>({{page.year}})</li>
+                                                                                        <!--href here should be {{paper.doilink}}-->            
+            <li class="article-doi"><span class="separator-disc">• </span><a target="_blank" href="#">Here'sGonnaBeTheDOILink</a></li>
```

Comment:
the DOI link and link description should also be pulled in from the posts files.